test/Microsoft.ML.Tokenizers.Tests/NormalizerTests.cs

```diff
@@ -129,6 +147,16 @@ public override string Normalize(string original)
 
                 return original.Normalize(_normalizationForm);
             }
+
+            public override string Normalize(ReadOnlySpan<char> original)
+            {
+                if (original.IsEmpty)
+                {
+                    return string.Empty;
+                }
+
+                return original.ToString().Normalize(_normalizationForm);
```

Comment:
Seems like we could `ReadOnlySpan<Char>.Normailze` if we wanted to save an allocation here.  Do you think that's common?

Reply:
Yes this is a test but I believe `ReadOnlySpan<Char>.Normailze` not exist too :-)